### ISSUE TITLE
Why the result of CycleGAN-tensorflow is so poor?

### ISSUE THREAD

**Issue by donghaoye** [XHUJOY/CycleGAN-tensorflow#5]
Why the result of CycleGAN-tensorflow is so poor?
It is far from the original CycleGAN.
I run it with 200 epoch

**Comment by XHUJOY** [comment]
sry, i do not have time to do this project now. Maybe you can use instance norm to replace batch norm used in this repo.

**Comment by donghaoye** [comment]
thks, could you give me some tips about  instance norm? or related paper and code ?

**Comment by XHUJOY** [comment]
batch norm has been replaced with instance norm in the newest commit
